content/en/synthetics/mobile_app_testing/settings/_index.md

```diff
@@ -61,6 +61,8 @@ To add a mobile application, navigate to the [**Mobile Applications List** tab][
 
 To edit or delete a mobile application, hover over a mobile application in the **Mobile Applications List** and click on the respective icon.
 
+**Note**: Mobile Application Testing does not provide full support for applications developed with Flutter SDK.
```

Comment:
Personal preference, but I feel Flutter SDK is not too commonly used. Recommended a simplified text below!
```suggestion
**Note**: Mobile Application Testing does not provide full support for Flutter applications.
```

Reply:
Applied! ✅